config.go

```diff
@@ -73,7 +73,7 @@ const (
 
 	// minTimeLockDelta is the minimum timelock we require for incoming
 	// HTLCs on our channels.
-	minTimeLockDelta = 4
+	minTimeLockDelta = routing.MinCLTVDelta
```

Comment:
Hmm, perhaps we should use a distinct constant for this value? This sets the min _CLTV delta_ allowed (when verifying against a users config values), while in this PR the main aim is to modify the min _final CTLV delta_ which governs the acceptable block delta for HTLC presented to the final hop in a route.

Reply:
> while in this PR the main aim is to modify the min final CTLV delta which governs the acceptable block delta for HTLC presented to the final hop in a route.

no, the goal is to clamp both forwarded and final CLTV deltas, which is the [change being made to the spec](https://github.com/lightningnetwork/lightning-rfc/pull/785). both will have a minimum recommended value of 18, there really isn't a reason to distinguish the two